src/main/resources/application.conf

```diff
@@ -6,8 +6,6 @@
   trapi-version = ${?TRAPI_VERSION}
   location = "http://localhost:8080"
   location = ${?LOCATION}
-  sparql-endpoint = "https://stars-app.renci.org/cam/sparql"
-  sparql-endpoint = ${?SPARQL_ENDPOINT}
```

Comment:
@jdr0887 why wasn't this method sufficient to get the SPARQL endpoint location from the environment? The docs make it seem like it would work: https://github.com/lightbend/config#optional-system-or-env-variable-overrides

Reply:
You are right.  I will make that change.